kytos/cli/commands/napps/parser.py

```diff
@@ -75,11 +76,15 @@ def parse_napps(napp_args):
 
     def parse_napp(arg):
         """Parse one argument."""
```

Comment:
Here we can use `napp_id` instead of `arg`.

And the method docstring need to be better written too.

By the way, why is `parse_napp()` inside `parse_napps()`? Any special reason for that?